pkg/chunk/storage/caching_index_client.pb.go

```diff
@@ -420,13 +420,8 @@ func (this *ReadBatch) String() string {
 	if this == nil {
 		return "nil"
 	}
-	repeatedStringForEntries := "[]Entry{"
-	for _, f := range this.Entries {
-		repeatedStringForEntries += strings.Replace(strings.Replace(f.String(), "Entry", "Entry", 1), `&`, ``, 1) + ","
-	}
-	repeatedStringForEntries += "}"
 	s := strings.Join([]string{`&ReadBatch{`,
-		`Entries:` + repeatedStringForEntries + `,`,
+		`Entries:` + strings.Replace(strings.Replace(fmt.Sprintf("%v", this.Entries), "Entry", "Entry", 1), `&`, ``, 1) + `,`,
```

Comment:
This change is removing https://github.com/gogo/protobuf/pull/557 - do we want to go backwards?

Reply:
There is no gogo release with that fix in it. 

We're not using the String function for any production use (I don't think displaying the proto on /ring is affected adversely.)